embedded-graphics/src/pixelcolor/mod.rs

```diff
@@ -108,5 +109,49 @@ pub trait PixelColor: Copy + PartialEq {
     /// See the [`raw` module documentation] for more details.
     ///
     /// [`raw` module documentation]: raw/index.html
-    type Raw: raw::RawData;
+    type Raw: RawData;
+}
+
+/// Convert a [`PixelColor`] into its underlying storage type
+///
+/// This trait provides the `into_storage()` method for implementors of [`PixelColor`]. This method
+/// exposes the underlying storage value of a pixel color type.
+///
+/// # Examples
+///
+/// ## Get the `u16` representing an `Rgb565` color
+///
+/// The [`Rgb565`] color type is stored in a `u16` internally. This example converts an [`Rgb565`]
+/// color into its underlying `u16` represenation.
+///
+/// ```rust
+/// use embedded_graphics::{prelude::*, pixelcolor::Rgb565};
+///
+/// let color = Rgb565::new(0xff, 0x00, 0xaa);
+///
+/// let raw: u16 = color.into_storage();
```

Comment:
Can this be misinterpreted that the type annotation is necessary? Perhaps adding an explicit type to the `0b..` constant below would be clearer.
```suggestion
/// let storage = color.into_storage();
```